src/APIActionCreator.es6.js

```diff
@@ -117,6 +117,11 @@ export default class APIActionCreator extends ActionCreator {
 			this
 		);
 
+		invariant(
+			withCredentials === undefined || typeof withCredentials === 'boolean',
```

Comment:
Won't this make every API action creator method require withCredentials? I feel like we should be defaulting to false if undefined.